blockchain/chain_sync/src/tipset_syncer.rs

```diff
@@ -1155,19 +1155,19 @@ async fn validate_tipset<DB: Blockstore + Store + Clone + Send + Sync + 'static,
 
     let mut validations = FuturesUnordered::new();
     for b in full_tipset.into_blocks() {
-        let validation_fn = tokio::task::spawn(validate_block::<_, C>(
-            consensus.clone(),
-            state_manager.clone(),
-            Arc::new(b),
-        ));
+        let consensus = consensus.clone();
+        let state_manager = state_manager.clone();
+        let validation_fn = tokio::task::spawn_blocking(move || {
```

Comment:
I can do some test but this might be the reason of low CPU utilization during tipset validation. a sync function scheduled with `spawn` will compete the async task pool which defaults to num of CPU cores, while using `spawn_blocking` allow it to use a different worker pool which defaults to 64

Reply:
I see, sure tests would be helpful. I'll move these changes to a follow up PR (leaving the scope of this PR to be just addition of tokio-console) as there might be more changes down the line.

Reply:
@hanabi1224 You're seeing low CPU utilization during tipset validation?